smoothscroll.js

```diff
@@ -78,8 +83,16 @@ document.addEventListener("DOMContentLoaded", function () {
     }
 });
 
+if (typeof exports !== "undefined") {
+  if (typeof module !== "undefined" && module.exports) {
+    exports = module.exports = smoothScroll;
+  }
+  exports.smoothScroll = smoothScroll;
+} else {
+  root.smoothScroll = smoothScroll;
+}
+
 // return smoothscroll API
 return smoothScroll;
 
-})();
-
+})(this);
```

Comment:
Nope you're right, I accidentally missed a step here. This should either be:
```
(function() {
var root = this;
...
}).call(this);
```

or
```
(function(root) {
...
})(this);
```

Which do you prefer? Either way I left this comment at the top:
```javascript
// In <script> tag context, `this` is the `window`.
// In Node.js context, `this` is the node `global`.
```

https://github.com/ahoym/smoothScroll/blob/support_modules_export/smoothscroll.js#L3-L4